packages/blade/src/storybook-recipes/SimpleCardWithTokens/styles.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Probably we can make this rule required only for exported modules

Reply:
Not sure if there's any rule like that exists.

Reply:
Non blocker: In short making it required only for [module boundary types](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md) and disabling it for other functions which aren't exported. Needs discussion within team though.